documentation/user_docs/docs/introduction/Download_and_setup.rst

```diff
@@ -169,34 +183,31 @@ use the ``hpc_config`` class.
 
-by inserting the location of the `Horace` and `Herbert` folders into these files,
-and copies them to a folder (``<extracted_folder>/ISIS`` by default) which it adds to the Matlab path
-by modifying the global ``pathdef.m`` file. 
-This means that all Matlab session including independent parallel workers have access to this path from any location where Matlab has been started.
-Unfortunately, this requires administrative (root) privileges.
-
-It is possible to install `Horace` without admin rights, in which case the ``horace_install`` script
-will create a ``pathdef.m`` file in the default `userpath` folder (as defined in the
-`Matlab documentation for search paths <https://uk.mathworks.com/help/matlab/matlab_env/what-is-the-matlab-search-path.html>`__).
-Some versions of Matlab, however, use different `userpath` folders if they are started as a parallel worker,
-which may lead the parallel extensions to fail because they cannot find the ``worker_v2.m`` file, but not to report any errors.
-In these cases, you should run Matlab from the `userpath` folder (e.g. the folder with the ``pathdef.m`` file).
-
-If instead of using the release packages you've cloned the `Horace` and `Herbert` repositories,
-then you should still run ``horace_install`` which is located in the ``admin`` subfolder of the Horace repository folder.
-However, you should now give the exact path to the `Horace` and `Herbert` folders using
-the ``horace_root`` and ``herbert_root`` arguments:
+by inserting the location of the `Horace` folders into these files, and copies them to a folder
+(``<extracted_folder>/ISIS`` by default) which it adds to the Matlab path by modifying the global ``pathdef.m`` file.
+This means that all Matlab session including independent parallel workers have access to this path from any location
+where Matlab has been started.  Unfortunately, this requires administrative (root) privileges.
+
+It is possible to install `Horace` without admin rights, in which case the ``horace_install`` script will create a
+``pathdef.m`` file in the default `userpath` folder (as defined in the `Matlab documentation for search paths
+<https://uk.mathworks.com/help/matlab/matlab_env/what-is-the-matlab-search-path.html>`__).  Some versions of Matlab,
+however, use different `userpath` folders if they are started as a parallel worker, which may lead the parallel
+extensions to fail because they cannot find the ``worker_v2.m`` file, but not to report any errors.  In these cases, you
+should run Matlab from the `userpath` folder (e.g. the folder with the ``pathdef.m`` file).
+
+If instead of using the release packages you've cloned the `Horace` repository, then you should still run
+``horace_install`` which is located in the ``admin`` subfolder of the Horace repository folder.  However, you should now
+give the path to the `Horace` folders using the ``horace_root`` and ``herbert_root`` arguments:
```

Comment:
this path setting is automatic for me - under what circumstances is it required?